pkg/app/piped/platformprovider/terraform/terraform_test.go

```diff
@@ -0,0 +1,36 @@
+package terraform
```

Comment:
Please add the file header (the copyright text)

Reply:
oh yea. forgot it. Thanks!